cdk.json

```diff
@@ -0,0 +1,70 @@
+{
+  "app": "npx tsx bin/app.ts",
+  "requireApproval": "never",
+  "output": ".cdk/cdk.out",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "tsconfig.json",
+      "package*.json",
+      "node_modules",
+      "test",
+      "bun.lockb"
+    ]
+  },
+  "context": {
```

Comment:
i'd appreciate a rundown or a link to some documentation on all these choices. I see ECS and EC2 in there which we don't use so just want to learn more about what they are doing

Reply:
Good question and I agree.  Unfortunately the cdk.json file is not very well documented, that file is very close to what you get for a cdk.json when you run:
```
cdk init app --language typescript
```
in a new directory to start a new project.  Throughout CDK's documentation they make references to change certain behaviors, change this or that in the cdk.json, I always start with the defaults the init provides.  I agree about EC2 and EC2, but I haven't tried removing them as they're there by default.  We could try, the CDK tooling may panic.